homeassistant/components/remote/__init__.py

```diff
@@ -38,8 +41,12 @@
 
 SCAN_INTERVAL = timedelta(seconds=30)
 SERVICE_SEND_COMMAND = 'send_command'
+SERVICE_SEND_COMMANDS = 'send_commands'
```

Comment:
The default implementation of send_commands on the abstract base class should be to just call `send_command` once for each command.